### PR TITLE
fix group search pipe DAT-39615

### PR DESCRIPTION
**Pull request by shaharkazaz**
This will only effect the select component

### CODE REVIEW

File: src/app/components/pipes/pipes-docs.component.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import { KababCaseExampleComponent } from './examples/kabab-case.ex';
+import KababCaseExampleCode from '!!raw-loader!./examples/kabab-case.ex';
+import { TranslateExampleComponent } from './examples/translate.ex';
+import TranslateExampleCode from '!!raw-loader!./examples/translate.ex';
+import { TrackByExampleComponent } from './examples/track-by.ex';
+import TrackByExampleCode from '!!raw-loader!./examples/track-by.ex';
+
+@Component({
+  template: `
+    <core-example title="Translate" [code]="TranslateExampleCode" [component]="TranslateExampleComponent">
```
Comment (NetanelBasal) [inline src/app/components/pipes/pipes-docs.component.ts]: 🔥🔥

File: projects/datorama/core/src/lib/sorting/search-results-group-sort.ts [hunk 2]
```diff
@@ -6,22 +6,58 @@
...
+import { findIndex } from '@datorama/utils';
 
 export interface SearchResultGroupSort<T> extends SearchResultSort<T> {
   groupLabel: string;
   groupKey: string;
 }
 
-export function searchResultsGroupSort<T = any>({ array = [], searchTerm = '', groupLabel = 'label', groupKey = 'children', labelKey = 'label' }: SearchResultGroupSort<T>): T[] {
+/**
+ * @param array
```
Comment (NetanelBasal) [inline projects/datorama/core/src/lib/sorting/search-results-group-sort.ts]: please remove JSDocs. (You can add to the IDE file that will not generate this)

File: src/app/components/pipes/pipes-docs.component.ts (hunk 1 above)
Comment (shaharkazaz) [reply src/app/components/pipes/pipes-docs.component.ts]: 😁

**Pull request merged**